lib/determinator/control.rb

```diff
@@ -81,6 +81,14 @@ def determinate(feature, id:, guid:, properties:)
 
       return feature.override_value_for(id) if feature.overridden_for?(id)
 
+      fixed_determination = choose_fixed_determination(feature, properties)
+      # Given constraints have specified that this actor's determination should be fixed
+      if fixed_determination
+        return false unless fixed_determination.on
+        return true unless feature.experiment?
+        return fixed_determination.variant
+      end
```

Comment:
The logic here doesn't seem right, correct me if I am wrong. Wouldn't we want to continue and check target groups if fixed determination is off? Same for if it's not an experiment?

Reply:
I think the on field is used to control a group of users that shouldnt see a feature in any circumstances rather then being a on/off flag for fixed determinations (meta!)

Reply:
Talking to @sherrry, I was thinking that maybe `feature_on` would be more understandable than `on`... What do you think? (Sorry, I suggested `on` in the first place!)